Taarafo.Core.Tests.Unit/Services/Foundations/Comments/CommentServiceTests.Exceptions.Add.cs

```diff
@@ -194,5 +194,50 @@ public async Task ShouldThrowServiceExceptionOnAddIfDatabaseUpdateErrorOccursAnd
             this.storageBrokerMock.VerifyNoOtherCalls();
             this.loggingBrokerMock.VerifyNoOtherCalls();
         }
+
+        [Fact]
+        public async void ShouldThrowValidationExceptionOnAddIfReferenceErrorOccursAndLogItAsync()
+        {
+            // given
+            DateTimeOffset randomDateTime =
+                GetRandomDateTimeOffset();
+
+            Comment randomComment = CreateRandomComment(randomDateTime);
+            Comment someComment = randomComment;
+            string randomMessage = GetRandomMessage();
+            string exceptionMessage = randomMessage;
+            var foreignKeyConstraintConflictException =
```

Comment:
Put a new line between multi-liners and one-liners

Reply:
@hassanhabib done